IncludeToolbox/Formatter/IncludeFormatter.cs

```diff
@@ -178,41 +176,40 @@ private static List<IncludeLineInfo> SortIncludes(IList<IncludeLineInfo> lines,
             else if (typeSorting == FormatterOptionsPage.TypeSorting.QuotedFirst)
                 sortedIncludes = sortedIncludes.OrderBy(x => x.LineDelimiterType == IncludeLineInfo.DelimiterType.Quotes ? 0 : 1);
 
-            // Finally, update the actual lines
+            // Merge sorted includes with original non-include lines
```

Comment:
Instead of doing what we were doing before to insert original non-include lines, this merges those in ahead of time. That way, there's less weird broken logic trying to do it on the fly :)